src/core/ui/forms/FormikMultiSelect.tsx

```
@@ -0,0 +1,114 @@
+      renderTags={(value, getTagProps) =>
+        value.map((option, index) => {
+          const isFixed = fixedOptions.some(fixed => fixed.id === option.id);
+
+          return (
+            <Chip
+              variant="outlined"
+              label={option.name}
+              {...getTagProps({ index })}
+              disabled={isFixed}
+              onDelete={isFixed ? undefined : getTagProps({ index }).onDelete}
+            />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)